index.html

```diff
@@ -0,0 +1,105 @@
+                </figure>
+                <h2>Starting with a staircase</h2>
+                <p>To start, we went back to a familiar pattern that Gattegno introduces in the textbooks, called a staircase. This is built from one rod of each colour arranged in size order (figure 1). The children were able to physically build the staircase. However, because they were so familiar with the pattern, they could also access it virtually, which would help with debugging and logical reasoning later. A role-play game to establish computational thinking was introduced before any actual programming began. This allowed the children to familiarise themselves with the function calls <strong>successor</strong> and <strong>predecessor (succ, pred)</strong> while traversing the staircase. The children would take turns to ask each other questions such as <strong>succ White</strong> (answer Red) or <strong>pred Orange</strong> (answer Blue). To help the children access the new programming vocabulary, one child</p>
+            </div>
+        </section>
+        <section>
+            <div>
+                <div>
+                    <figure>
+                        <img src="../img/2.jpg" alt="math">
+                        <figcaption>Transcript of a conversation about program behaviour.</figcaption>
+                    </figure>
+                </div>
+                <div>
+                    <p>would take the role of the ‘engine’ and the other ‘interpreter’. This was extended to <span>succ</span> $ <span>succ</span> (successor of successor) which uses function composition $. Some children could creatively enjoy inventing functions such as <span>succ</span> $ <span>succ</span> $ <span>succ</span> <span>White</span> to test the technique. The children experimented by writing down their calls before building themselves a rod calculator in Haskell. The language soon became a natural part of lessons and meant that it was easy to progress to the next step.</p>
+                    <p>run, and that this would require carefully debugging and retyping. The children could spot bugs in the engine because they had already worked out the answer. The staircase was a familiar pattern, so they could predict what the output should be. Children were also able to edit the <span>Colour</span> data type to model odd or even staircases and test their functions to see what the effect would be on the derived <span>successor</span> function. A transcript (see insert) shows a pupil working with the</p>
+                </div>
+                <div>
+                    <span>"I</span>
+                    <blockquote>
+                        <p>some children could creatively enjoy inventing functions</p>
+                    </blockquote>
+                </div>
+                <div>
+                    <h2>Programming</h2>

```

Comment:
this h2 and following next p tag makes sense to wrap into separate container since in future we will need to put them in one left column and the last p - to the right column 
![image](https://user-images.githubusercontent.com/104353194/166154527-55bbc0c4-7ec3-43d2-8a3d-d35dd03e20e2.png)